histomicstk/features/ComputeFSDs.py

```diff
@@ -0,0 +1,179 @@
+import collections
+import numpy as np
+from skimage.segmentation import find_boundaries
+
+
+def ComputeFSDs(Label, K=128, Fs=6):
+    """
+    Calculates `Fourier shape descriptors` from a label mask.
+
+    Parameters
+    ----------
+    Label : array_like
+        A M x N label mask.
```

Comment:
Yes. That is what I'm going to do. Will enable the ComputeFSDs.py (for example) to receive a labeled image and to perform the loops for each object.